#Pan/Tilt fault status
#None = 0
#PanJointLocked = 1
#TiltJointLocked = 2
Header header
uint64 value
